haskell-backend/src/main/java/org/kframework/backend/haskell/HaskellBackend.java

```diff
@@ -41,9 +46,13 @@ public void accept(Backend.Holder h) {
         files.saveToKompiled("definition.kore", kore);
         ProcessBuilder pb = files.getProcessBuilder();
         List<String> args = new ArrayList<>();
-        args.add("kore-parser");
-        args.add("--no-print-definition");
+        args.add("kore-exec");
         args.add("definition.kore");
+        args.add("--module");
+        args.add(kompileOptions.mainModule(files));
+        args.add("--output");
+        args.add("haskellDefinition.bin");
+        args.add("--serialize");
```

Comment:
I assume this option implies "don't run the thing yet"?

Reply:
Yes, --serialize means no actual execution takes place.